### PR TITLE
Heightmap (rendering only)

### PR DESCRIPTION
**Pull request by chapulina**
Requires:

* ~~https://github.com/osrf/sdformat/pull/388 (that targets `sdf9`, I've been using the equivalent `chapulina/10/heightmap` branch to test this)~~
* ~~https://github.com/ignitionrobotics/ign-rendering/pull/180~~

~~This works for me when compiling Ogre 1 from source (tag 1.9.1), but not when using debs. With debs, the heightmap only shows if the heightmap is created during `RenderUtil::Init` **as well as** its current place. I'm still investigating.~~ Fixed on `ign-rendering`.

~~This also needs to be updated as I change the `ign-rendering` implementation so heightmap inherits from geometry.~~ Done

This is working now:

image

This is ready for review.

### CODE REVIEW

File: examples/worlds/heightmap.sdf [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+      filename="ignition-gazebo-user-commands-system"
+      name="ignition::gazebo::systems::UserCommands">
+    </plugin>
+    <plugin
+      filename="ignition-gazebo-scene-broadcaster-system"
+      name="ignition::gazebo::systems::SceneBroadcaster">
+    </plugin>
+    <plugin
+      filename="ignition-gazebo-contact-system"
+      name="ignition::gazebo::systems::Contact">
```
Comment (iche033) [inline examples/worlds/heightmap.sdf:30]: is this system needed?

Comment (chapulina) [reply examples/worlds/heightmap.sdf]: Nope, removed

**Review by iche033 — COMMENTED** [review]
code changes look good. Just waiting for green CI

**Comment by iche033** [comment]
there are some compile errors in windows CI for other unrelated classes. Looks like it's happening in other PRs too.

**Comment by chapulina** [comment]
> there are some compile errors in windows CI 

That's been fixed :tada: (ignore the test failures :upside_down_face: )

**Review by iche033 — APPROVED** [review]
🎉

**Pull request merged**